src/main/resources/docs/patterns.json

```diff
@@ -368,7 +379,7 @@
       "default" : ".*"
     }, {
       "name" : "exceptionVariableName",
-      "default" : "^$"
+      "default" : "\"^$\" (empty)"
```

Comment:
This seems wrong. The `(empty)` part of the string should probably be removed

Reply:
Now the DocGenerator is fixed and parses only the real regular expression and not the comments (like the `(empty)` you see here.